source/locates.html.md

```diff
@@ -0,0 +1,295 @@
+| TotNoOrders | 68 |  | Total number of list order entries across all messages |  | Numeric |  | R | N |
+| NoOrders | 73 |  | Number of orders in this message |  | Numeric |  | R | N |
+| ClOrdID | 11 |  | Unique ID for the Order. Must be the first in the repeatable list. |  | AlphaNumeric | 25 | R | Y |
+| ListSeqNo | 67 |  | Order number within this list |  | Numeric |  | R | Y |
+| Symbol | 55 |  | Security ticker | AAPL | Alpha |  | R | Y |
+| Side | 54 | 1-Buy;2-Sell;3-Buy Minus;4-Sell Plus;5-Sell Short;6-Sell Short Exempt;7-Undisclosed;8-Cross;9-Cross Short | Side | 1 | Numeric | 1 | O | Y |
+| Text | 58 |  | Any comments by trader | comment | AlphaNumeric |  | O | Y |
+| TransactTime | 60 |  | Date+Time in UTC | 20220121-13:27:43.000 | YYYYMMDD-HH:mm:SS:sss | 21 | R | Y |
+| OrdType | 40 | 1-Market;2-Limit;3-Stop;4-Stop Limit;5-Market On Close;6-With or Without;7-Limit or Better;8-Limit With or Without;9-On Basis;A-On Close;B-Limit On Close;C-Forex C;D-Previously Quoted;]E-Previously Indicated;F-Forext F;G-Forex G;H-Forex H;I-Funari;P-Pegged | Order Type | 1 | AlphaNumeric | 1 | O | Y |
+| ClientID | 109 |  | Firm ID/MPID | CLST | AlphaNumeric |  | R | Y |
+| ExecBroker | 76 |  | Credential string: username,password | USRNM,PSWD | AlphaNumeric |  | R | Y |
+| OrderQty | 38 |  | Number of share ordered | 1000 | Numeric |  | R | Y |
+| IDSource | 22 | 1-CUSIP;2-SEDOL;4-ISIN;8-Exchange Symbol | Security ID type | 1 | Numeric | 1 | CR | Y |
+| SecurityID | 48 |  | Security ID as per IDSource tag 22 | 037833100 | AlphaNumeric | 12 | CR | Y |
+| ClearingAccount | 440 |  | Trader ID | TRDRID | AlphaNumeric |  | O | Y |
+| Account | 1 |  | Account ID | ACCTID | AlphaNumeric |  | R | Y |
+
+
+### Execution Report (Request)
+```
+8=FIX.4.2\u00019=59\u000135=8\u000134=28512\u000149=CLSTLOCT\u000152=20220208-21:10:45.336\u000156=OMSC\u000137=200004\u000111=12345671\u000155=IBM\u000154=1\u000160=20220208-21:10:00.792\u0001109=ACCOUNTID\u000138=1000\u000158=THISISASINGLELOCATERESPONSEBYSYMBOL\u000117=200004\u000120=0\u0001150=B\u000139=B\u00011=ACCOUNTID\u0001151=0\u000114=1000\u00016=23.00\u000144=0.23\u000110=025\u0001
+```
+`Execution Report` (One Response Message for each item in List of Order Request) is a response sent by Clear Street with a locate. 
+There will be one response message for each item if the response is for a list of multiple securities request via `New Order - List`
+
+| Field Name | FIX Tag # | Possible Values | Comments | Example | Format | Length | Required |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| MsgType | 35 | 8 | Message type | 8 | AlphaNumeric | 1 | R |
+| OrderID | 37 |  | Unique Locate ID |  | AlphaNumeric |  | R |
+| ClOrdID | 11 |  | ClOrdID from the New Order message |  | AlphaNumeric | 25 | R |
+| Symbol | 55 |  | Security ticker | AAPL | Alpha |  | R |
+| Side | 54 | 1-Buy;2-Sell;3-Buy Minus;4-Sell Plus;5-Sell Short;6-Sell Short Exempt;7-Undisclosed;8-Cross;9-Cross Short | Side | 1 | Numeric | 1 | R |
```

Comment:
I think all of the Possible values should sit on their own line... it is pretty hard to read on my screen in the rich text mode

Reply:
Thanks. Made the change using <br/>. Didn't know how to do that earlier.